### DISCUSSION TITLE
Character count with/without space character inversed?

### DISCUSSION THREAD

**Discussion by georgd** [Zettlr/Zettlr#2514] [General]
Before I write a ticket, I want to make sure I understand this correctly. Looking at the word and character count, I noticed that the higher char count number reads "ohne Leer" (without space). Shouldn’t that be the number with space character?

![image](https://user-images.githubusercontent.com/214002/134692036-c79ad682-ed2d-44d5-8457-dec923b64445.png)

**Comment by nathanlesage** [comment]
There is already an issue open, I have that on my radar!

**Reply by georgd** [reply]
Oh, thank you! Sorry for bothering you.
